sycl/test-e2e/Matrix/get_coord_bf16_matA_impl.hpp

```diff
@@ -175,7 +175,7 @@ int main() {
 
   for (int i = 0; i < MATRIX_M; i++) {
     for (int j = 0; j < MATRIX_K; j++) {
-      A[i][j] = i;
+      A[i][j] = j;
```

Comment:
What is the reason for this change? If the goal is to init matrix with different values to improve verification, how about this?
```suggestion
      A[i][j] = i+j;
```

Reply:
Done!